### PR TITLE
Add llvm-binary-utilities to .github/CODEOWNERS

### PR DESCRIPTION
**Pull request by MaskRay**
From https://discourse.llvm.org/t/input-needed-teams-for-pull-request-subscriptions/73116/3

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -617,3 +617,33 @@ utils/bazel/llvm-project-overlay/libc/** @llvm/pr-subscribers-libc
 /clang/test/CodeGen/WebAssembly/ @llvm/pr-subscribers-webassembly
 /clang/test/Sema/*wasm* @llvm/pr-subscribers-webassembly
 /clang/test/SemaCXX/*wasm* @llvm/pr-subscribers-webassembly
+
+# llvm-binary-utilities
+/llvm/docs/CommandGuide/llvm-* @llvm/pr-subscribers-llvm-binary-utilities
```
Comment (MaskRay) [inline .github/CODEOWNERS:622]: Seems that https://github.com/orgs/llvm/teams/pr-subscribers-llvm-binary-utilities is present?

Comment (JDevlieghere) [reply .github/CODEOWNERS:622]: This looks like the issue we ran into for lldb too. Mehdi explained it here: https://discourse.llvm.org/t/input-needed-teams-for-pull-request-subscriptions/73116/89

Comment (MaskRay) [reply .github/CODEOWNERS:622]: I know, but I got a warning when the team is actually present:)

**Comment by MaskRay** [comment]
Merging this as we have some changes that do not have reviewers set (e.g. some llvm-nm patches). I think this should be fixed soon.

**Comment by MaskRay** [comment]
It seems that llvm-binary-utilities cannot be added as reviewers.

**Pull request merged**